utils/scarb-ui/src/lib.rs

```diff
@@ -140,7 +140,9 @@ impl Ui {
 
     /// Print a warning to the user.
     pub fn warn(&self, message: impl AsRef<str>) {
-        self.print(TypedMessage::styled("warn", "yellow", message.as_ref()))
+        if self.verbosity > Verbosity::NoWarnings {
```

Comment:
Why `>` and not `>=`? 🤔

Reply:
The goal here is to avoid printing warnings when verbosity is set to `NoWarnings`. 

Using `> Verbosity::NoWarnings` directly communicates that intent, unlike using `>= Verbosity::Normal` for this purpose.

Reply:
Right, sorry :D